src/systemtest/resources/features/ArtifactFlow.feature

```diff
@@ -4,13 +4,53 @@ Feature: Artifact system test
   @ArtifactFlowScenario
   Scenario: Artifact Flow Scenario
```

Comment:
I would want to have a short summary what the scenario does. You may add it in multi line comment under the scenario: tag, no " or * or anything needed. it till be marked as just white text in a proper terminal when executing the test.